src/amo/sagas/home.js

```diff
@@ -76,12 +76,12 @@ export function* fetchHomeAddons({
       page_size: LANDING_PAGE_THEME_COUNT,
     },
   };
-  const trendingExtensionsParams: SearchParams = {
+  const popularExtensionsParams: SearchParams = {
```

Comment:
I wonder if there is any easier way to toggle this.

Reply:
I agree that this is a pain, and there is probably a better way, but we keep saying that this is "temporary" and we'll have better tools to manage this stuff in the future, so we just persist with the manual "search and replace" approach for now. I have done a couple of things over the past year or so that have made incremental improvements, but we still have to make a lot of changes when we want to change one of the shelves.